pytest_djangoapp/fixtures/__init__.py

```diff
@@ -6,7 +6,7 @@
 from .mail import mail_outbox
 from .messages import messages
 from .request import request_factory, request_get, request_post, request_client
-from .settings import settings
+from .settings import app_name, djapp_options, settings
```

Comment:
`settings` here refres to sjango settings.
Let's move plugin specific stuff into `.plugin`.